databricks/koalas/groupby.py

```diff
@@ -98,26 +99,44 @@ def aggregate(self, func_or_funcs, *args, **kwargs):
         1  1  0.589
         2  3  0.705
 
+        >>> aggregated = df.groupby('A').agg({'B': ['min', 'max'], 'C': 'sum'})
```

Comment:
Can we give a shot with `OrderedDict` and see if it works?

Reply:
got it!

Reply:
or we can drop `'C': 'sum'` here because the first doctest already covers it.

Reply:
Yea, looks we can do that too.